presto-kafka-stream/kafkastream.properties

```diff
@@ -0,0 +1,3 @@
+connector.name=kafkastream
```

Comment:
What is the purpose of this file here?

Reply:
I didn't add this in presto-main, so for now I am keeping this here as an example. once connector is included, will remove from here.